components/footer/Contacts.tsx

```diff
@@ -0,0 +1,97 @@
+import { FC, useState } from 'react';
+
+// import Link from 'next/link';
+
+import {
+  Box,
+  Button,
+  Collapse,
+  Divider,
+  List,
+  ListItemButton,
+  ListItemText,
+  useMediaQuery,
+} from '@mui/material';
+
+const developerNames = ['Daniel Silva', 'Santiago Salamanca', 'Camilo Cortes'];
+
+const githubs = [
+  'https://github.com/DANIELSSF',
+  'https://github.com/SantiagoSalamancaa',
+  'https://github.com/CamiloCortesM',
+];
+
```

Comment:
<!-- Inside the "constants" folder, create a file named "developerData.ts" -->

example
**File: constants/developerData.ts**

```typescript
export const DEVELOPER_DATA = [
  {
    name: 'Daniel Silva',
    github: 'https://github.com/DANIELSSF'
  },
  {
    name: 'Santiago Salamanca',
    github: 'https://github.com/SantiagoSalamancaa'
  },
  {
    name: 'Camilo Cortes',
    github: 'https://github.com/CamiloCortesM'
  }
];